leaflet-distance-marker.js

```diff
@@ -31,12 +31,33 @@ L.DistanceMarkers = L.LayerGroup.extend({
 		var cssClass = options.cssClass || 'dist-marker';
 
 		var zoomLayers = {};
-		var length = L.GeometryUtil.length(line);
+		// Get line coords as an array
+        	  if (typeof line.getLatLngs == 'function') {
+                	var coords = line.getLatLngs();
```

Comment:
Good spot. In this case it should sets coords to line. 
var coords = line;

I will update.